src/routers/api/v1/posts.ts

```diff
@@ -22,7 +29,11 @@ router.post("/", koaBody(), async ctx => {
     await getManager().transaction("SERIALIZABLE", async transactionalEntityManager => {
         var user = ctx.state.token.user
         const res = await transactionalEntityManager.increment(User, { id: user.id }, "postsCount", 1)
-        user = await transactionalEntityManager.findOneOrFail(User, user.id)
+        user = await transactionalEntityManager.findOneOrFail(
+            User,
+            { id: user.id },
+            { relations: ["avatarFile", "avatarFile.variants"] }
```

Comment:
variants はここで引くと何かしら不都合があったはずなのでavatarFileだけにしておいたほうがよさそう (pack時に無いと引かれるので)
```suggestion
            { relations: ["avatarFile"] }
```